app/domain/output_document.rb

```diff
@@ -33,15 +33,15 @@ def guider_organisation
   end
 
   def value_of_pension_pots
-    return 'No value given' if appointment_summary.value_of_pension_pots.blank?
-
-    pension_pot = number_to_currency(appointment_summary.value_of_pension_pots, precision: 0)
-
-    if appointment_summary.upper_value_of_pension_pots.blank?
-      pension_pot
-    else
-      pension_pot + ' to ' + \
-        number_to_currency(appointment_summary.upper_value_of_pension_pots, precision: 0)
+    pension_pot = to_currency(appointment_summary.value_of_pension_pots)
+    upper_limit = to_currency(appointment_summary.upper_value_of_pension_pots)
```

Comment:
These variable names could be a bit confusing in the future to someone not quite so absorbed as we currently are in this domain.

`approximation` not `approximate`?

We take an upper value then call it a limit? Maybe `upper_value`? Then maybe `lower_value`?